dali/python/nvidia/dali/external_source.py

```diff
@@ -397,6 +474,29 @@ def __call__(self, *, source = None, cycle = None, name = None, layout = None, c
         if batch is None:
             batch = True
 
+        if prefetch_queue_depth is None:
```

Comment:
Why not make `prefetch_queue_depth` default to 1?

Reply:
So:
```
prefetch_queue_depth = self._prefetch_queue_depth or prefetch_queue_depth 
```

Reply:
I think it is the approach taken by other arguments. We set everything as a None, select one taken from init or call. If we adjust that, we should do it for all arguments.

Reply:
Makes sense

Reply:
It's to handle the specification in two places. I will add error checking for doulbe spec.